README.md

```diff
@@ -1,5 +1,9 @@
 > :warning: This fork is intended for use in liferay-portal only. Our fork of YUI is deprecated and only critical patches will be applied case by case.
 
+> :warning: Use `npm install` followed by `npm run build` to build this project since we have overridden the original build from YUI3.
+
+> :warning: See [Confluence](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/1965457449/Releasing+YUI+AUI) for instructions on how to release a new version.
```

Comment:
Should we just copy/paste that section here and add a reference to this link? Thinking just in case we migrate away from confluence in the future(like grow, loop, etc)

Reply:
You mean there are changes we go back to loop again? :trollface:

Reply:
I thought about that too, but decided not to do it to avoid duplication, in case someone changes confluence but not this readme, for example :shrug: